server/webapp/localize.xml

```diff
@@ -157,6 +157,9 @@
     <entry key="ENV_ADD_FAILED">Failed to add environment. {0}</entry>
     <entry key="ENV_UPDATE_FAILED">Failed to update environment ''{0}''. {1}</entry>
     <entry key="ENV_DELETE_FAILED">Failed to delete environment ''{0}''. {1}</entry>
+    <entry key="CONFIG_REPO_ADD_FAILED">Failed to add config repository. {0}</entry>
```

Comment:
@varshavaradarajan -- there are couple of specific messages that we need to get rid of and use the generic message,
I will create a new issue for refactoring of localize message and will send a PR to fix it.